src/gui/tray/Window.qml

```diff
@@ -851,7 +851,55 @@ ApplicationWindow {
             anchors.right: trayWindowMainItem.right
         }
 
+        Loader {
+            id: newActivitiesButtonLoader
+            anchors.top: activityList.top
+            anchors.topMargin: 5
+            width: Style.newActivitiesButtonWidth
+            height: Style.newActivitiesButtonHeight
+            anchors.horizontalCenter: activityList.horizontalCenter
+            z: 1
+
+            active: false
+
+            sourceComponent: CustomButton {
+                id: newActivitiesButton
+                hoverEnabled: true
+                padding: Style.smallSpacing
+
+                textColor: Style.currentUserHeaderTextColor
+                textColorHovered: Style.currentUserHeaderTextColor
+                contentsFont.bold: true
+                bgNormalColor: Qt.lighter(bgHoverColor, 1.25)
+                bgHoverColor: Style.currentUserHeaderColor
+                bgNormalOpacity: 0.8
+                bgHoverOpacity: 1.0
+
+                anchors.fill: parent
+
+                text: qsTr("New activities")
+
+                icon.source: "image://svgimage-custom-color/expand-less-black.svg" + "/" + Style.currentUserHeaderTextColor
+                icon.width: Style.activityLabelBaseWidth
+                icon.height: Style.activityLabelBaseWidth
+
+                onClicked: {
+                    activityList.scrollToTop();
+                    newActivitiesButtonLoader.active = false;
+                }
+
+                Timer {
+                    id: disappearTimer

```

Comment:
Nitpick but, IMO, QML's ids offer a good opportunity to explicitly name what a component is explicitly for (and more explicit names make it much easier to understand what other components are asking for when referencing the id elsewhere

```suggestion
                    id: newActivitiesButtonDisappearTimer

```